imsv-docs-astro/src/content/docs/guides/core-concepts/card-funding.mdoc

```diff
@@ -42,41 +42,60 @@ of digital assets needed to meet a users desired spend in their desired fiat
+1. Obtaining an Immersve-generated signature permitting a given withdrawal
+2. Executing a withdrawal and observing the effect on the funding source
 
-If the user has not transacted using the solution before then the [get spending
-prerequisites](/api-reference/get-spending-prerequisites) response will
-typically call for an ERC20 `approve` in favour of the smart contract followed
-by a `depositAndLock` invocation of the smart contract.
+To begin the withdrawal process your application will
+[create a withdrawal intent](https://docs.immersve.com/api-reference/create-withdrawal-intent/),
+providing the amount of the intended withdrawal and a reference to the relevant funding source.
 
-If multiple transactions are present then they should be carried out in the
-order in which they are presented.
+Prior to returning a response, Immersve will identify the current blockchain nonce for the cardholder.
+The nonce is used to ensure only the intended withdrawal intent can be executed by the cardholder.
+The returned withdrawal intent is valid for the period indicated by the `expiresAt` property.
+
+When a withdrawal intent is created, the relevant funding source balance is debited by the amount of
+the funding source interaction. The funding source interaction `status` will be "pending" at this time
+and will remain so until such time as the withdrawal intent is executed.
 
-The client application is to parse and formulate the raw blockchain transaction
-message such that it is presented to the user within the web3 wallet and signed
-and sent to the relevant blockchain.
+The cardholder may create further withdrawal intents to change the amount of the withdrawal prior
+to having executed it. If there are multiple withdrawal intents created with the same nonce, the
+funding source will be reduced by the amount of the highest withdrawal intent.
 
+To execute a withdrawal the cardholder must submit the transaction contained in the `execution` property
+of the withdrawal intent prior to the `expiresAt` timestamp of the intent. Your application should prompt
+the cardholder to sign and submit the transaction. Upon observation of the the execution of the withdrawal
+intent on the blockchain, Immersve will update the `status` of the funding source interaction.
+Your application should query the interaction to observe the change to the `status` and make this known to
+the cardholder accordingly.
+
+<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764578946735882&cot=14 -->
+![Card Funding Sequence](@assets/diagrams/withdrawal-sequence.svg)
+
+
+### Currency Conversion
+
+A user may be quoted a price for a purchase by a merchant in a local fiat
+currency. To determine the sufficient amount of local fiat currency to fund a
+card in its billing currency (USD), use the [currency
+conversion](/api-reference/currency-conversion) API endpoint.
+
+The returned value can passed to the get spending prerequisites endpoint as
```

Comment:
```suggestion
The returned value can be passed to the get spending prerequisites endpoint as
```

Reply:
also would be good to add a link to prerequisites endpoint docs

Reply:
Thanks Kira, fixed!